content/zh-cn/blog/_posts/2023-04-06-keeping-kubernetes-secure-with-updated-go-versions.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: blog
+title: "使用更新后的 Go 版本保持 Kubernetes 安全"
+date: 2023-04-06
+slug: keeping-kubernetes-secure-with-updated-go-versions
+---
+<!--
+layout: blog
+title: "Keeping Kubernetes Secure with Updated Go Versions"
+date: 2023-04-06
+slug: keeping-kubernetes-secure-with-updated-go-versions
+-->
+
+<!--
+**Author**: [Jordan Liggitt](https://github.com/liggitt) (Google)
+-->
+
+**作者**：[Jordan Liggitt](https://github.com/liggitt) (Google)
+
+**译者**：顾欣 (ICBC)
+
+### 问题 {#the-problem}
+
+<!--
+Since v1.19 (released in 2020), the Kubernetes project provides 12-14 months of patch releases for each minor version.
+This enables users to qualify and adopt Kubernetes versions in an annual upgrade cycle and receive security fixes for a year.
+-->
+
```

Comment:
This blank line (line 28) is not necessary.

You can delete this kind of blank lines from start to end.